### ISSUE TITLE
Create pre-publication review checklist for all content types

### ISSUE THREAD

**Issue by rinkug** [betterscientificsoftware/betterscientificsoftware.github.io#391]
Before the document can go online and be considered ready for publication, we need to have a checklist of the things that need to be reviewed in the document before we can push it out onto the website. This document is for figuring out what this checklist will look like. 

Please note that the editorial processes will have a shepherd/reviewers for all stages who will review the topic being proposed, the actual content being written and whether it fits the BSSw content policies etc. This current document is more about what auxillary things need to be in place for the document to be considered ready for publication.

Before we can develop review checklists for various types of content, we need to have a clear idea of the types of content we expect to support. Our original understanding of our content types was described in the What to Contribute document. Lately, there is another PR/issue (PR #373, Issue #361) in the works which attempts to revise the content types and their attributes. These proposed changes are listed in the content types and their attributes document. We will follow this new document for our checklist discussion.

DRAFT PR can be found here: [#392](https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/pull/392). There may be additional discussion in the PR.

When you have finished with your commentary, etc. on this topic, please check your name off the following list. If I have missed your name, please add it to the list

- [ ] @rinkug 
- [x] @markcmiller86 
- [ ] @pagrubel 
- [ ] @prwolfe 
- [ ] @curfman 
- [ ] @bernhold 
- [ ] @bartlettroscoe 
- [ ] @JeanShuler 
- [ ] @jarrah42

**Comment by markcmiller86** [comment]
So, here are some links to previous comments/issues on this topic I am aware of...

- Several comments in [this pull request](https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/pull/372)
- Several comments and some external resources in [this issue](https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/issues/371)
- Finally, there are some minimal criteria defined [here](https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/blob/master/Site/CuratedContentEditorialWorkflow_Alt.md#light-weight-process-for-authors) in our current CC workflow doc

**Comment by rinkug** [comment]
> So, here are some links to previous comments/issues on this topic I am aware of...
> 
> * Several comments in [this pull request](https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/pull/372)
> * Several comments and some external resources in [this issue](https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/issues/371)
> * Finally, there are some minimal criteria defined [here](https://github.com/betterscientificsoftware/betterscientificsoftware.github.io/blob/master/Site/CuratedContentEditorialWorkflow_Alt.md#light-weight-process-for-authors) in our current CC workflow doc

Thanks Mark. I have added the few things proposed in these discussions. The PR is now online and can be reviewed.

**Comment by rinkug** [comment]
Closing this issue since only the PR #392 is sufficient for discussion
